### ISSUE TITLE
Add option to online application form

### ISSUE THREAD

**Issue by dennisbaaten** [internetstandards/Internet.nl-dashboard#527]
Our policy has changed. 
We now also allow API access for "for-profit organizations that offer services to (a substantial amount of) non-profit organizations or to organizations in sectors of high criticality." See also number 7 at https://github.com/internetstandards/Internet.nl-API-docs/tree/main/application_form.

Please add this option to the online form at https://dashboard.internet.nl/#/signup

**Comment by stitch** [comment]
![image](https://github.com/user-attachments/assets/0a7c0ea8-c748-4042-a398-47e906ac0766)

zit in de volgende versie, het is wel een wat lange tekst zo. bedoel je niet 'kritieke ketenpartner'?

**Comment by dennisbaaten** [comment]
Nee, in de NIS2 context wordt er gesproken over vitaal (categorie A). En partijen die vanwege het verlenen van diensten aan een vitale partij aan specifieke vereisten moeten voldoen, mogen ook API toegang krijgen.

**Comment by dennisbaaten** [comment]
Ik zou de laatste zin als volgt formuleren: "Commerciële organisatie die diensten verleent aan een (substantieel aantal) non-profit organisaties en/of organisaties die zijn aangemerkt als vitale infrastructuur (categorie A; NIS2)"
